include/nrf_rpc/nrf_rpc_cbkproxy.h

```diff
@@ -6,26 +6,26 @@
 
 /**
  * @file
- * @defgroup bt_rpc_cbkproxy Bluetooth RPC callback proxy API
```

Comment:
By this move, the declarations in this file become a part of the public API. Then it probably requires a release note (now or when contributing to main).

Reply:
I guess it should be done in PR to main. There are no release notes for 2.8.0 yet, since 2.7.0 has not been released yet.